src/api/finishBuild.ts

```diff
@@ -66,7 +73,13 @@ export async function finishBuild({token, ...body}: Params) {
         title,
         summary: `
 ${galleryUrl ? `[View Image Gallery](${galleryUrl})` : ''}
-
+${
+  terminationReason === 'maxChangedSnapshots'
+    ? '# Max number of changed snapshots exceeded (snapshot run was terminated early)'
+    : terminationReason
+    ? `# This run was terminated early with termination reason: ${terminationReason}`
+    : ''
+}
```

Comment:
Oh yeah, you are right 🤔 Any ideas why this would not show up there? I cant really test this without running the PR each time, so it's annoying to figure out

Reply:
Moved it to text field and going to try again